src/DataBox/readme.md

```diff
@@ -123,6 +123,10 @@ directive:
           - IdentityType
           - DeliveryType  
           - Detail
+          
+  - from: source-file-csharp
+    where: $
+    transform: $ = $.replace('internal Microsoft.Azure.PowerShell.Cmdlets.DataBox.Models.Api20210301.IJobSecrets', 'public Microsoft.Azure.PowerShell.Cmdlets.DataBox.Models.Api20210301.IJobSecrets');  
```

Comment:
can you help understand why this need to be public?

Reply:
Hey @VeryEarly , Utiltity of Get-AzDataBoxJobCredential operation is to help customer in fetching the secrets, if this is not public then how he will see secrets